mmv1/products/dialogflowcx/api.yaml

```diff
@@ -560,3 +560,75 @@ objects:
           - :RUNNING
           - :SUCCEEDED
           - :FAILED
+  - !ruby/object:Api::Resource
+    name: 'Environment'
+    base_url: "{{parent}}/environments"
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      Represents an environment for an agent. You can create multiple versions of your agent and publish them to separate environments.
+      When you edit an agent, you are editing the draft agent. At any point, you can save the draft agent as an agent version, which is an immutable snapshot of your agent. 
+      When you save the draft agent, it is published to the default environment. When you create agent versions, you can publish them to custom environments. You can create a variety of custom environments for testing, development, production, etc.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/dialogflow/cx/docs'
+      api: 'https://cloud.google.com/dialogflow/cx/docs/reference/rest/v3/projects.locations.agents.environments'
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 60
+          update_minutes: 60
+          delete_minutes: 60
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: parent
+        url_param_only: true
+        input: true
+        description: |
+          The Agent to create an Environment for. 
+          Format: projects/<Project ID>/locations/<Location ID>/agents/<Agent ID>.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: |
+          The name of the environment. Format: projects/<Project ID>/locations/<Location ID>/agents/<Agent ID>/environments/<Environment ID>
```

Comment:
This is no longer the correct format of this field after the custom flatten is applied.

Reply:
you are right I removed the format